internal/gonut/cf/cf.go

```diff
@@ -289,6 +304,27 @@ func getApp(appName string) (*AppDetails, error) {
 	return cfCurlAppByGUID(appGUID)
 }
 
+// getAppRoute returns the public URL of the application
+// using its name and the Cloud Foundry host domain.
+func getAppRoute(appName string) (string, error) {
+	domain, err := getDomain(appName)
+	if err != nil {
+		return "", err
+	}
+
+	return fmt.Sprintf("http://%s.%s", appName, domain), nil
+}
+
+// getAppStatusCode sends a GET request to the given URL
+// and returns the statuscode.
+func getAppStatusCode(appRoute string) (int, error) {
+	resp, err := http.Get(appRoute)
```

Comment:
One thing to investigate a little: Even though we do not use the response body, do we still need to add a `defer resp.Body.Close()` to make sure we close everything?